### PR TITLE
Include actions for Environment Destruction

### PR DESCRIPTION
**Pull request by thisisthetechie**
## Include actions for Environment Destruction

Related issue: [CB2-10415](https://dvsa.atlassian.net/browse/CB2-10415)

## Checklist
- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: .github/workflows/tf-destroy.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
         run: |
           terraform init -lock-timeout=300s
           terraform workspace select develop
-          echo "## Terraform Initialised" >> $GITHUB_STEP_SUMMARY
+          echo "## Terraform Initialized" >> $GITHUB_STEP_SUMMARY
```
Comment (hamiltonrosscgi) [inline .github/workflows/tf-destroy.yaml:66]: American spelling?

File: .github/workflows/build-mobile.yaml [hunk 2]
```diff
@@ -117,8 +117,7 @@ jobs:
           bundle exec ${{ inputs.fastlane_cmd }}
 
       - name: ⬆️ Store QR Code
-        if: ${{ inputs.is_auto == false }}
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v3
```
Comment (hamiltonrosscgi) [inline .github/workflows/build-mobile.yaml]: Should be v4?

File: .github/workflows/tf-destroy.yaml (hunk 1 above)
Comment (thisisthetechie) [reply .github/workflows/tf-destroy.yaml:66]: Yeah, it reads right to me - initialised means "to use initials" in my head - like "Pete Mallam initialised is PM".

You want it changing?

Comment (hamiltonrosscgi) [reply .github/workflows/tf-destroy.yaml:66]: Initialised is valid UK spelling, just wondered on the reason for change :) happy either way.

**Review by hamiltonrosscgi — APPROVED** [review]

**Review by JoshCarter-ops — APPROVED** [review]

**Pull request merged**